var/spack/repos/builtin/packages/nmap/package.py

```diff
@@ -15,6 +15,7 @@ class Nmap(AutotoolsPackage):
     url = "https://nmap.org/dist/nmap-7.70.tar.bz2"
     list_url = "https://nmap.org/dist-old/"
 
+    version("7.93", sha256="a5479f2f8a6b0b2516767d2f7189c386c1dc858d997167d7ec5cfc798c7571a1")
```

Comment:
`spack checksum nmap 7.93` yields `55bcfe4793e25acc96ba4274d8c4228db550b8e8efd72004b38ec55a2dd16651`, not the hash provided.